src/main/java/utils/Code.java

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+import lombok.Getter;
+import lombok.Setter;
+import model.User;
+
+public class Code {
+
+    @Getter
+    private String code;
+
+    @Setter
+    @Getter
+    private User user;
```

Comment:
better add Code code in user model
this class shouldn't depend from user

Reply:
Solved.